x-pack/plugins/cloud_security_posture/public/pages/compliance_dashboard/compliance_dashboard.tsx

```diff
@@ -231,10 +251,14 @@ const determineDashboardDataRefetchInterval = (data: ComplianceDashboardDataV2 |
   return false;
 };
 
-const TabContent = ({ posturetype }: { posturetype: PosturePolicyTemplate }) => {
+const TabContent = ({
```

Comment:
I found this change a bit confusing and counter-intuitive, I think it would make more sense to have the parent handle the redirect, for example, adding the Redirect component in the `ComplianceDashboard` component return function:

```jsx
return (
<DashboardTabRedirecter>
    <CloudPosturePage>
      //....
    </CloudPosturePage>
</DashboardTabRedirecter>
  );
```

Reply:
DashboardTabRedirecter return `Redirect` components do you mean adding Redirect compoent  to `CloudPosturePage`?

Reply:
Reverted this change after getting race condition.